src/components/container/MyDataHelpsAssistant/MyDataHelpsAssistant.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+import { faPaperPlane } from '@fortawesome/free-solid-svg-icons/faPaperPlane';
+import { faUser } from '@fortawesome/free-solid-svg-icons/faUser';
+import { faGear } from '@fortawesome/free-solid-svg-icons/faGear';
+import { faFlask } from '@fortawesome/free-solid-svg-icons/faFlask';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons/faSpinner';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons/faChevronDown';
+import { StreamEvent } from '@langchain/core/tracers/log_stream';
+import { AIMessageChunk } from '@langchain/core/messages';
+import { StructuredTool } from '@langchain/core/tools';
+import Markdown from 'react-markdown';
+import { MyDataHelpsAssistant } from '../../../helpers/assistant/assistant';
+
+import '@fortawesome/fontawesome-svg-core/styles.css';
+import "./MyDataHelpsAssistant.css";
+
+export interface MyDataHelpsAssistantProps {
```

Comment:
This is MyDataHelpsUI so the MyDataHelps prefix is redundant; how about "AIAssistant" for the name of this thing